src/HeaderBag.php

```diff
@@ -23,7 +23,10 @@ public static function fromHeaderLines(array $lines)
 
         foreach ($lines as $line)
         {
-            list($name, $value) = explode(': ', $line, 2);
```

Comment:
No guarantee that explode would return two values, and that space is optional but not ignored if present – slightly too much there to. Unfortunately I think that's slightly too conditional for a nice one liner, would be pleased if wrong though!